pkg/tnf/interactive/spawner.go

```diff
@@ -272,6 +281,23 @@ func NewGoExpectSpawner() *GoExpectSpawner {
 	return &GoExpectSpawner{}
 }
 
+// LogStdErr logs stderr output to logger (warning)
+func LogStdErr(cmdLine string, stderrpipe io.Reader) {
+	buf := bufio.NewReader(stderrpipe)
+
+	for {
+		line, _, err := buf.ReadLine()
+		log.Warn("STDERR for " + cmdLine + " : " + string(line))
```

Comment:
why not using logrus.Error ?

Reply:
the output of stderr might or might not be an error for our main program. With warning I was trying to say that it could be a error

Reply:
for instance:
WARN[0072] STDERR for exec -n tnf -it test-697ff58f87-c8vcv -c test -- sh : Unable to use a TTY - input is not a terminal or the right kind of file

Reply:
I also prefer Warning here.